downloader/src/main/kotlin/Downloader.kt

```diff
@@ -56,7 +59,7 @@ import org.ossreviewtoolkit.utils.withoutPrefix
 /**
  * The class to download source code. The signatures of public functions in this class define the library API.
  */
-object Downloader {
+class Downloader(private val config: DownloaderConfiguration = DownloaderConfiguration()) {
```

Comment:
The other similar classes (`Analyzer`, `Scanner`) don't assign a default value for their configurations; so why do we here?

Reply:
Because I thought it's nicer to call `Downloader()` instead of `Downloader(DownloaderConfiguration())`.
But to keep consistency I'll change it to the way the Analyzer and Scanner are doing it.